docs/src/dev-guide/coding-guidelines.md

```
@@ -53,6 +53,110 @@ To avoid including a state variable in a React Hook's dependency array, you can
+    exportLogs: () => {
+      const logExportManager = new LogExportManager();
+      set({logExportManager});
+      
+      const {exportProgress} = get();
+      // If name doesn't match, use colon to separate the variable name from the value.
+      set({exportProgress: EXPORT_LOGS_PROGRESS_VALUE_MIN});
+    },
+}));
+```
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)